content/en/docs/quickstart.md

```diff
@@ -29,19 +43,39 @@ docker build -t localhost:5000/net-monitor:v1 https://github.com/wabbit-networks
 docker push localhost:5000/net-monitor:v1
 ```
 
+Save the digest value of the image from the output of the `docker push` command.
+
+*Important*: Always use the digest value of an image when signing since they are immutable. Tag values are mutable and can reference a different container image than the original signed container image.
+
+An example output of `docker push`:
+
+```output
+The push refers to repository [localhost:5000/net-monitor]
+2556c54bfdf3: Pushed
+fb6ca4f9c8d3: Pushed
+ded7a220bb05: Pushed
+v1: digest: sha256:073b75987e95b89f187a89809f08a32033972bb63cda279db8a9ca16b7ff555a size: 942
+```
+
+In the above example, the reference to the container image using the digest value is `localhost:5000/net-monitor@sha256:073b75987e95b89f187a89809f08a32033972bb63cda279db8a9ca16b7ff555a`.
+
+> Note：
+> If you use a tag value to sign an image, `notation` will determine the digest value of the current image associated and will use that digest to sign. Tags can be used to reference the container image.
+
 ## List the signatures associated with the container image
 
 Use `notation list` to show any signatures associated with the container image you built and pushed in the previous section.
 
 ```console
+IMAGE=localhost:5000/net-monitor@sha256:073b75987e95b89f187a89809f08a32033972bb63cda279db8a9ca16b7ff555a
 notation list $IMAGE
 ```
 
 Confirm there are no signatures shown in the output.
 
 ## Generate a test key and self-signed certificate
 
-Use `notation cert generate-test` to generate a test RSA key for signing artifacts, and a self-signed X.509 test certificate for verifying artifacts.
+Use `notation certificate generate-test` to generate a test RSA key for signing artifacts, and a self-signed X.509 test certificate for verifying artifacts.
```

Comment:
To keep consistency and simplicity, I would suggest using the short alias by default for all CLI commands in Notation docs and examples.

Reply:
@zr-msft @toddysm any thoughts on suggestion? I don't have strong opinions.

Reply:
@FeynmanZhou @yizha1  as long as we stay consistent, I'm fine with short alias or full length